package.json

```diff
@@ -23,13 +23,11 @@
     "build": "aegir build",
     "test": "aegir test -t node -t browser --no-cors",
     "test:node": "aegir test -t node",
-    "test:browser": "aegir test -t browser -t webworker --no-cors",
-    "test:node": "aegir test -t node",
+    "test:browser": "aegir test -t browser --no-cors",
```

Comment:
Why remove the webworker test?

Reply:
@diasdavid It came in when I did a rebase.  Should I restore to original?